src/primitives/block.h

```diff
@@ -92,7 +92,7 @@ class CBlock : public CBlockHeader
     ADD_SERIALIZE_METHODS;
 
     template <typename Stream, typename Operation>
-    inline void SerializationOp(Stream& s, Operation ser_action, int nType, int _nVersion) {
+    inline void SerializationOp(Stream& s, Operation ser_action, int nType, int nVersion) {
```

Comment:
I think I figured this out, see https://github.com/bitcoin/bitcoin/pull/8658#issuecomment-250096029 and the comment below that.